csrankings-9.csv

```diff
@@ -1693,6 +1694,7 @@ Youjian Zhao,Tsinghua University,http://www.tsinghua.edu.cn/publish/csen/4623/20
 Youjip Won,KAIST,http://oslab.kaist.ac.kr/professor-youjip-won,qy7uO-UAAAAJ
 Youming Qiao,University of Technology Sydney,https://www.uts.edu.au/staff/youming.qiao,rfr7X5IAAAAJ
 Youn-Long Steve Lin,National Tsing Hua University,http://www.cs.nthu.edu.tw/~ylin,jYgzZWIAAAAJ
+Youn-kyung Lim,KAIST,http://cixd.kaist.ac.kr,49PRJ1EAAAAJ

```

Comment:
Apparently in Industrial Engineering; please provide evidence that this person meets the inclusion criteria (sole advisor of PhD in _Computer Science_). Not listed here: https://cs.kaist.ac.kr/people/faculty